doc/src/site/sphinx/ml/sw_xgboost.rst

```diff
@@ -118,18 +118,23 @@ XGBoost Memory Configuration
 H2O XGBoost uses additionally to Java memory, off-heap memory. This means that it requires some additional memory
 available on the system.
 
-When running on YARN, please make sure to set the ``memoryOverhead`` so XGBoost has enough memory. On Spark, the following
-properties might be set
+When running on YARN or Kubernetes, please make sure to set the ``SUBST_EXECUTOR_MEMORY_OVERHEAD`` so XGBoost has enough
+native memory on executors. It's recommended to set the property to 12O% of the value set in ``spark.executor.memory``.
 
-- ``spark.yarn.am.memoryOverhead`` - in case of YARN Cluster deployment
-- ``spark.yarn.driver.memoryOverhead`` - in case of YARN client and other deployments
-- ``spark.yarn.executor.memoryOverhead`` - in all deployment scenarios
+Note: ``SUBST_EXECUTOR_MEMORY_OVERHEAD`` must be set in MiB.
 
-On YARN, the container size is determined by ``application_memory * memory_overhead``. Therefore, by specifying the
-overhead, we are also allocating some additional off-heap memory which XGBoost can use.
+Example
+```````
+If you set ``spark.executor.memory`` to ``10g``, ``SUBST_EXECUTOR_MEMORY_OVERHEAD`` should be set to ``12288``.
+The size of the corresponding YARN or Kubernetes container will be at least 22 GiB.
 
-In Spark Standalone Mode or IBM Conductor environment, please make sure to configure the following configurations:
+Note: In case of Pysparkling, the YARN container will be bigger about the memory required by the Python process.
 
 
-- ``spark.memory.offHeap.enabled=true``
-- ``spark.memory.offHeap.size=4G`` - example of setting this property to 4G of off-heap memory
```

Comment:
@jakubhava IMHO, this option has a different meaning. This option limits Spark itself when doing manual memory allocation to the offheap space. E.g.: Tungsten memory representation, `df.persist(persist(StorageLevel.OFF_HEAP)`

Wouldn't ``spark.executor.memoryOverhead`` work for Spectrum Conductor as well.  I don't see it in [the list of unsupported parameters](https://www.ibm.com/support/knowledgecenter/en/SSZU2E_2.4.1/reference_s/spark_properties_unsupported.html). 

@FengAtH2O what option do you for XGBoost and specifying  the memoryOverhead (off-heap) space on Spectrum Conductor?